### PR TITLE
Split donate from the sponsor page

### LINKED ISSUE #299: Add Donate link to navbar
Currently the line is a bit blurry between donate and sponsor - best to split it into two pages so that we can have a bit of blurb for donations too

### PR DESCRIPTION
**Pull request by antoinepetty**
Fixes #299

Not sure about this text, happy to take alternative suggestions. 

Similarly, the order of the links in the navbar/footer could be changed

Navbar/footer changes will be added to docs and competition website once this is approved

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -24,6 +24,12 @@ <h4>&nbsp;</h4>
         <li><a href="{{ '/contact/' | prepend:site.baseurl }}">Contact us</a></li>
       </ul>
     </div>
+    <div class="column d-4-12 m-2-12">
+      <h4>&nbsp;</h4>
+      <ul class="sitemap">
+        <li><a href="https://link.justgiving.com/v1/charity/donate/charityId/3118662?tipScheme=TipJar2.1&isRecurring=true&amount=20.00&reference=givingcheckout_tj21">Donate</a></li>
```
Comment (PeterJCLaw) [inline _includes/footer.html:30]: Now that we have this url in several places, could we hoist it to a constant somewhere? Perhaps in the config?

**Comment by antoinepetty** [comment]
I don't know what to do with some of the FAQ questions.

e.g. 

> How can I donate?
>There are 2 options for donating;
>
>1) You can use the JustGiving link above. This will also allow you to add Gift Aid. If >you wish to be recognised as a Bronze, Silver, Gold, or Platinum sponsor please >also contact us using the above form so we can easily identify your donation.
>
>2) Contact us using the above form and we can provide details for a bank >transfer.

**Comment by PeterJCLaw** [comment]
What was the motivation for this change (PR)? It feels like this split is both making the FAQ a bit harder to explain and also taking up more space in the navigation. (Not that that's necessarily an issue, though space there is limited)

I suspect better understanding the motivation for the split could help us work out what to do for the FAQ entry.

**Comment by antoinepetty** [comment]
I'm not sure how universal this point of view is but I feel that donating and sponsoring are quite distinct things. Sponsoring would be a company giving us money in return for advertising. Donating would be individuals giving us money out of goodwill.

Part of Diane's original vision for the website was a separate Donate page but we didn't think we had enough content to justify a full page instead of linking directly to justgiving.

(+sorry for lazily requesting your review on a bit of an unfinished pr)

**Comment by PeterJCLaw** [comment]
> I'm not sure how universal this point of view is but I feel that donating and sponsoring are quite distinct things. Sponsoring would be a company giving us money in return for advertising. Donating would be individuals giving us money out of goodwill.
> 
> Part of Diane's original vision for the website was a separate Donate page but we didn't think we had enough content to justify a full page instead of linking directly to justgiving.

Ah, that makes sense.
(In future it would be great to see this sort of detail included in the description and/or the commit messages -- understanding the "why" of changes is really useful & important!)

Since this page is then meant just for sponsors, perhaps reworking the question to be more like this could work?

> *How do I become a sponsor?"
> Use the form above to get in touch. Please include a summary of who you are, what you're able to offer us and which level you have in mind. We'll get in touch (usually within a few days) 

> (+sorry for lazily requesting your review on a bit of an unfinished pr)

No worries, I kinda drive-by reviewed previously.
Not sure how incomplete this is though -- maybe having set it as a "Draft" might have been a clearer way to show that it was still incomplete? Alternatively if it's something you're stuck on or would like non-review input on, perhaps specifying that would be useful too? (I've just created the "help wanted" label which has been used for this sort of thing in other repos, not sure why this repo didn't have one)

**Comment by antoinepetty** [comment]
Thanks for the wording, that does seem to finalise this page as sponsor-only.

Would be good to get some feedback on the ordering of the nav buttons if possible. It feels weird to have donate on the end but I can't explain why.
image

> (In future it would be great to see this sort of detail included in the description and/or the commit messages -- understanding the "why" of changes is really useful & important!)

That's a really good point so I'll make note of that for the future

> > (+sorry for lazily requesting your review on a bit of an unfinished pr)
> 
> No worries, I kinda drive-by reviewed previously. Not sure how incomplete this is though -- maybe having set it as a "Draft" might have been a clearer way to show that it was still incomplete? Alternatively if it's something you're stuck on or would like non-review input on, perhaps specifying that would be useful too? (I've just created the "help wanted" label which has been used for this sort of thing in other repos, not sure why this repo didn't have one)

It was just this FAQ section which I saw as unfinished but I should probably have put this in draft mode after noticing it!

File: index.html [hunk 2]
```diff
@@ -166,6 +166,17 @@ <h2>Our sponsors</h2>
     </div>
   </div>
 </div>
+<div class="panel panel-thin">
+  <div class="panel-container">
+    <div class="fixed-width text-center">
+      <h2>Donate</h2>
+      <p>Every donation, no matter the amount, contributes to us running the competition for our teams.</p>
```
Comment (sedders123) [inline index.html:173]: More friendly?
```suggestion
      <p>Every donation is greatly appreciated, no matter the amount, and directly contributes to us running the competition for our teams.</p>
```

**Review by sedders123 — APPROVED** [review]

**Review by sedders123 — APPROVED** [review]

**Review by raccube — APPROVED** [review]

**Pull request merged**